app/components/SwapDetails.tsx

```diff
@@ -6,19 +6,17 @@ type SwapDetailsProps = {
 };
 
 // TODO: Hack, GET /swaps response should format decimals like GET /orders ?
-function insertDecimals(num, dec) {
-  return (num / 10 ** dec).toFixed(dec);
-}
+// function insertDecimals(num, dec) {
+//   return (num / 10 ** dec).toFixed(dec);
+// }
 
 // TODO: Hack, asset name should be returned by GET /swaps ?
-function getAssetName(n) {
-  if (n === 'hbit') return 'BTC';
-  return 'DAI';
-}
-
-export default function SwapDetails(props: SwapDetailsProps) {
-  const { properties } = props;
+// function getAssetName(n) {
```

Comment:
You can remove unused code.

Reply:
I'll action on that later once I touch the swap pages.